source/_integrations/reolink.markdown

```diff
@@ -72,6 +72,13 @@ If an entity listed below has an asterisk (*) next to its name, it means it is d
 If an entity listed below has a plus (+) next to its name, it means this entity supports push updates. These entities will have almost instant state changes. 
 For redundancy, the state of all entities is also polled every 60 seconds. For entities without a plus (+), this is the only update method. Therefore, a device's state change can take up to 60 seconds to be reflected in Home Assistant.
 An exception is the firmware update entity, which is polled every 12 hours.
+Another exception are battery cameras, most enties are still polled every 60 seconds. However, the entities that would cause the camera to wake from sleep will only be polled during the following events:
```

Comment:
```suggestion
Another exception are battery cameras, most {% term entities %} are still {% term polling polls %} every 60 seconds. However, the entities that would cause the camera to wake from sleep will only be polled during the following events:
```